logprep/run_logprep.py

```diff
@@ -60,6 +58,9 @@ def cli() -> None:
     Logprep allows to collect, process and forward log messages from various data sources.
     Log messages are being read and written by so-called connectors.
     """
+    if "pytest" not in sys.modules:  # needed for not blocking tests
+        signal.signal(signal.SIGTERM, signal_handler)
+        signal.signal(signal.SIGINT, signal_handler)
```

Comment:
How about we add a test that checks if these signals actual work as promised? Like running logprep with `process = subprocess.Popen()`, `process.send_signal()` and then checking if the process actually terminated as expected?

Reply:
I tried this, but it did not work. 
I guess the reason is, that the logprep process name if started with subprocess is not "MainProcess". So the triggers do not work.

Reply:
too bad. but good to know that you tried it already, thanks! :+1: